README.md

```diff
@@ -58,11 +58,12 @@ To further configure your Datadog construct, use the following custom parameters
-| `forwarderARN`         | When set, the plugin will automatically subscribe the functions' log groups to the Datadog Forwarder.  |
-| `flushMetricsToLogs` | Send custom metrics using Cloudwatch logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter, it's required to set the parameters `site` and `apiKey` (or `apiKMSKey` if encrypted). |
-| `site`               | Set which Datadog site to send data, only needed when flushMetricsToLogs is `false`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`. |
-| `apiKey`             | Datadog API Key, only needed when `flushMetricsToLogs` is `false`. For more information about getting a Datadog API key, see the [API key documentation][8]. |
-| `apiKMSKey`          | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false`, and you are using KMS encryption. |
+| `extensionLayerVersion`   | Version of the Datadog Lambda Extension layer to install, such as 5. When `extensionLayerVersion` is set, `site` must be set, and `apiKey` (or `apiKMSKey` if encrypted) need to be set. While using `extensionLayerVersion` do not set `forwarderARN`.The Datadog Lambda Extension layer is in public preview. Learn more about the lambda extension [here][12].   |
+| `forwarderARN`         | When set, the plugin will automatically subscribe the functions' log groups to the Datadog Forwarder. Do not set `forwarderARN` when `extensionLayerVersion` is set.|
+| `flushMetricsToLogs` | Send custom metrics using Cloudwatch logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter or set `extensionLayerVersion`, it's required to set the parameters `site` and `apiKey` (or `apiKMSKey` if encrypted). |
+| `site`               | Set which Datadog site to send data, only needed when flushMetricsToLogs is `false` or `extensionLayerVersion` is set. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`. |
+| `apiKey`             | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set. For more information about getting a Datadog API key, see the [API key documentation][8]. |
+| `apiKMSKey`          | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `extensionLayerVersion` is set, and you are using KMS encryption. |
 | `enableDDTracing`    | Enable Datadog tracing on your Lambda functions. Defaults to `true`. When enabled, it's required to set the `forwarderARN` parameter. |
```

Comment:
I added the warnings above multiple times in each of the related parameters, I read it multiple times and could not think of a better way of writing it. Im open to suggestions on this.

Reply:
I think that makes sense, people might just look up one specific parameter, and they should see the warning for that parameter.